src/commands/staff/lookup.ts

```diff
@@ -42,18 +49,29 @@ export default class extends Command {
 			]);
 
 		if (!entry.pii) {
-			const sender: SageUser = await msg.client.mongo.collection(DB.USERS).findOne({ discordId: msg.author.id });
-			msg.channel.send(`That user has not opted in to have their information shared over Discord. 
-	An email has been sent to you containing the requested data.`);
-			this.sendEmail(sender.email, member.user.username, entry);
+			const sender: SageUser = await interaction.client.mongo.collection(DB.USERS).findOne({ discordId: user.id });
+			interaction.reply({ content: `That user has not opted in to have their information shared over Discord. 
+	An email has been sent to you containing the requested data.`, ephemeral: true });
+			this.sendEmail(sender.email, user.username, entry);
 			return;
 		}
 
-		msg.author.send({ embeds: [embed] }).then(() => msg.channel.send('I\'ve sent the requested info to your DMs'))
-			.catch(() => msg.channel.send('I couldn\'t send you a DM. Please enable DMs and try again'));
+		interaction.user.send({ embeds: [embed] }).then(() => interaction.reply({
```

Comment:
Wouldn't this just be easier sending the info as an ephemeral message rather than a dM?

Reply:
Good call